psalm.xml

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="7"
-    totallyTyped="true"
-    resolveFromConfigFile="true"
```

Comment:
going with the default strictness level of 2. The stricter level 1 is too hardcore with too many `Mixed*` error messages: https://psalm.dev/docs/running_psalm/error_levels/

Reply:
Sounds good for me!